src/argilla/training/setfit.py

```diff
@@ -26,12 +26,11 @@ class ArgillaSetFitTrainer(ArgillaTransformersTrainer):
     _logger = logging.getLogger("ArgillaSetFitTrainer")
     _logger.setLevel(logging.INFO)
 
-    require_version("torch")
-    require_version("datasets")
-    require_version("transformers")
-    require_version("setfit>=0.6")
-
     def __init__(self, *args, **kwargs):
+        require_version("torch")
+        require_version("datasets")
+        require_version("transformers")
+        require_version("setfit>=0.6")
```

Comment:
I can work on a PR to extend the `require_version` and `requires_version` decorator to accept also lists of dependencies rather than just one by one, as I think it would be more readable